### PR TITLE
fix rake version per logstash rakelib/vendor.rake

### PR DESCRIPTION
**Pull request by colinsurprenant**

### CODE REVIEW

File: logstash-devutils.gemspec [hunk 1]
```diff
@@ -21,7 +21,10 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 2.14.0" # MIT License
   spec.platform = "java"
   spec.add_runtime_dependency "jar-dependencies" # MIT License
-  spec.add_runtime_dependency "rake" # MIT License
+
+  # make sure this rake version is in sync with the logstash rakelib/vendor.rake rake version
```
Comment (jordansissel) [inline logstash-devutils.gemspec]: I would add a parenthetical `(the one that JRuby ships with)`

Comment (jordansissel) [inline logstash-devutils.gemspec]: Unless this is just to hardcode an upgrade.. I forget my brain sometimes.

**Comment by jordansissel** [comment]
LGTM

**Comment by colinsurprenant** [comment]
better explanation comment now?

**Comment by jordansissel** [comment]
hmm-thumbs-up

**Comment by colinsurprenant** [comment]
WOO :D
:shipit:

**Comment by colinsurprenant** [comment]
so.... since we decided on another strategy in elasticsearch/logstash#2475  this change is not required anymore and I think I'd prefer to leave the rake version unspecified :P

**Pull request closed without merging**